File: platforms/web/examples/core/texture/aspect.js

import { Renderer } from "fragmentcolor";

const renderer = new Renderer();
// 1x1 RGBA (white) raw pixel bytes
const pixels = new Uint8Array([255,255,255,255]);
const tex = await renderer.createTexture(pixels, { size: [1, 1] });
const a = tex.aspect();

File: platforms/kotlin/examples/texture/texture/aspect.kt

import org.fragmentcolor.*

val renderer = Renderer()
val bytes: ByteArray = byteArrayOf()
// 1x1 RGBA (white) raw pixel bytes
val pixels = listOf(255.0f, 255.0f, 255.0f, 255.0f)
val tex = renderer.createTexture(TextureInputMobile.Bytes(pixels.let { ba -> ByteArray(ba.size) { i -> ba[i].toInt().and(0xFF).toByte() } }), null)
val a = tex.aspect()